pkg/registry/apps/replicaset/strategy_test.go

```diff
@@ -49,11 +50,7 @@ func TestReplicaSetStrategy(t *testing.T) {
 			ObjectMeta: metav1.ObjectMeta{
 				Labels: validSelector,
 			},
-			Spec: api.PodSpec{
-				RestartPolicy: api.RestartPolicyAlways,
-				DNSPolicy:     api.DNSClusterFirst,
-				Containers:    []api.Container{{Name: "abc", Image: "image", ImagePullPolicy: "IfNotPresent", TerminationMessagePolicy: api.TerminationMessageReadFile}},
-			},
+			Spec: podtest.MakePod("").Spec,
```

Comment:
Why not MakePodSpec?  I am not sayin git should or shouldn't but it seems inconsistent to me to use it in some places and not others.

I won't repeat this comment, but "same everywhere else" :)

Reply:
So one thing is that sometimes it is easier to do MakePod("").Spec vs MakePodSpec(api.RestartPolicyAlways).
Going to see if defaulting the RestartPolicy is a better alternative.